typify-impl/src/test_util.rs

```diff
@@ -104,18 +104,16 @@ fn validate_output_impl<T: JsonSchema + Schema>(ignore_variant_names: bool) {
 /// is reasonable to expect that the generated Rust binding matches exactly the Rust type from
 /// which the JSON Schema was generated.
 ///
-/// The code generation logic that generates Rust bindings given JSON Schema always canonicalizes
+/// Our code generation logic, always canonicalizes
 /// any standard type (eg, `Option` is output as `::std::option::Option`), to avoid potential
 /// conflicts with types in the JSON schema with conflicting names like `Option`.  Unfortunately,
-/// this complicates the test cases that start with a Rust type that implements `JsonSchema`, use
-/// that to generate a JSON schema for that type, then use typify to generate a Rust binding for
-/// that type, and expects that the Rust AST for the source type and the generated type are exactly
-/// the same.
+/// this complicates the test cases that validate that a round-trip from Rust to JSON back to Rust
+/// are exactly the same.
```

Comment:
wrap, remove comma

Reply:
I rewrapped and removed the extraneous comma.  

I rebased against upstream `main` to resolve the conflict with `Cargo.toml`.

Hopefully this is good to go now...